From 4126dd51e90deb9e22dfd139cc4518a7812fcad6 Mon Sep 17 00:00:00 2001
From: Yorhel <git@yorhel.nl>
Date: Fri, 29 Oct 2021 14:29:45 +0200
Subject: uit_dl.c: Use mvaddstr() instead of mvprintw() for dynamic string

Trips up on Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=997167

Not actually a bug in this case, the given string cannot contain printf
formatting codes.
---
 src/uit_dl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/uit_dl.c b/src/uit_dl.c
index 2012b49..dd75144 100644
--- a/src/uit_dl.c
+++ b/src/uit_dl.c
@@ -224,7 +224,7 @@ static void t_draw(ui_tab_t *tab) {
   if(sel) {
     char hash[40] = {};
     base32_encode(sel->hash, hash);
-    mvprintw(bottom, 0, hash);
+    mvaddstr(bottom, 0, hash);
   } else
     mvaddstr(bottom, 0, "Nothing selected.");
   mvprintw(bottom, wincols-19, "%5d files - %3d%%", g_hash_table_size(dl_queue), pos);
-- 
cgit v1.2.3

